### PR TITLE
ref(app-platform): remove SentryAppWebhookError model from state

### PR DESCRIPTION
**Pull request by nolachen**
Step 2 of removing SentryAppWebhookError: Remove the model from Django state

I accidentally created+merged this same PR but the base branch was not master (#15528)

This PR actually merges it to master. Only merge + deploy after #15523 is deployed

### CODE REVIEW

**Review by wedamija — APPROVED** [review]
LGTM, I wonder if you edit the base branch on the other PR to be master if it would let you merge again.

**Comment by nolachen** [comment]
@wedamija Tried to but couldn't edit the base branch after the PR is marked as merge

**Pull request merged**
